idb/postgres/postgres_migrations.go

```diff
@@ -538,3 +528,56 @@ func MakeDeletedNotNullMigration(db *IndexerDb, state *MigrationState) error {
 	}
 	return sqlMigration(db, state, queries)
 }
+
+// MaxRoundAccountedMigration converts the import state.
+func MaxRoundAccountedMigration(db *IndexerDb, migrationState *MigrationState) error {
+	db.accountingLock.Lock()
+	defer db.accountingLock.Unlock()
+
+	nextMigrationState := *migrationState
+	nextMigrationState.NextMigration++
+
+	f := func(ctx context.Context, tx *sql.Tx) error {
+		defer tx.Rollback()
+
+		importstate, err := db.getImportState(tx)
+		if err == idb.ErrorNotInitialized {
+			// Leave uninitialized.
+			db.log.Printf("Import state is not initialized, leaving unchanged.")
+		} else if err != nil {
+			return err
+		} else {
+			if importstate.AccountRound == nil {
```

Comment:
I think you can move this to `if err == idb.ErrorNotInitialized || importstate.AccountRound == nil {` and avoid the extra indent. It probably ends up in that block anyway.

Reply:
No, this would be a bug if `err != nil` and `err != idb.ErrorNotInitialized`!